src/scala/tiefvision-web/app/core/ImageProcessing.scala

```diff
@@ -24,7 +24,8 @@ object ImageProcessing {
   def randomSmilarityImage = SimilarityImages(random.nextInt(SimilarityImages.length))
 
   def findSimilarImages(image: String, finderFolder: String, imagesFolderOpt: Option[String] = None): ImageSearchResult = {
-    val luaSearch = s"luajit search.lua $image ${imagesFolderOpt.getOrElse("")}"
+    val luaConfigFlag = Option(sys.props("luaConfig")).fold("") { path => s"-config $path" }
```

Comment:
I would rather do pattern matching, as for me it's more clear, but OK if you want to keep like that.